src/Sulu/Bundle/CoreBundle/Command/WebspacesInitCommand.php

```diff
@@ -158,6 +159,11 @@ private function setBasicLocalizationProperties(
             $node->setProperty($this->properties->getName('nodeType'), Structure::NODE_TYPE_CONTENT);
         }
 
+        if (!$node->hasProperty($this->properties->getName('nodeType'))) {
+            // set title
+            $node->setProperty($this->properties->getName('title'), 'Sulu CMF');
```

Comment:
Does this set the title for all localizations? And shouldn't we use something more general like `Homepage`?